lib/SiftResponse.php

```diff
@@ -27,9 +35,17 @@ public function __construct($result, $httpStatusCode, $request) {
             }
 
             if (is_null($this->body)) {
-                $this->apiErrorMessage = 'Invalid JSON received from SiftScience API';
+                $this->apiErrorMessage = 'Invalid JSON received from Sift API';
             } elseif (is_array($this->body)) {
-                // NOTE: Responses from /v3 endpoints don't contain status or error_message.
+                // /v3 responses use error and description
```

Comment:
I added new fields with names prefixed with `api` to match the existing `/v2xx` error fiends with suffixes matching the `/v3` response fields. I don't like that we have two sets of error fields, but they don't map one-to-one with each other, and I'd rather keep them mapping cleanly to the raw (and documented) response bodies.